Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -124,9 +124,31 @@ class BrowserViewController: UIViewController {
     }
 
     override func traitCollectionDidChange(previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
         updateToolbarState()
         showToolbars(animated: false)
-        super.traitCollectionDidChange(previousTraitCollection)
+    }
+
+    override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransitionToSize(size, withTransitionCoordinator: coordinator)
+
+        /**
+        *  WKWebView looks like it has a bug where it doesn't invalidate it's visible area when the user
+        *  performs a device rotation. Since scrolling calls
+        *  _updateVisibleContentRects (https://github.com/WebKit/webkit/blob/master/Source/WebKit2/UIProcess/API/Cocoa/WKWebView.mm#L1430)
+        *  this method nudges the web view's scroll view by a single pixel to force it to invalidate.
+        */
```

Comment:
Nit: `*` alignment (or just use `//` since this isn't a documentation comment anymore).